benchmarks/src/main.rs

```diff
@@ -129,7 +124,9 @@ fn main() {
         run_jobs::<sudoku::Job>(&cli.out, sudoku::new_jobs());
     }
 
-    if cli.command == Command::All || cli.command == Command::Zeth {
+    // TODO: re-enable once zeth gets updated
+    // if cli.command == Command::All ||
+    if cli.command == Command::Zeth {
```

Comment:
Disabled zeth from the benchmark until we figure how to properly patch or zeth gets updated to a compatible version

Reply:
Using an updated branch on zeth worked